lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -144,6 +144,17 @@ describe('CategoryRenderer', () => {
 
     const audits = categoryDOM.querySelectorAll('.lh-audit');
     assert.equal(audits.length, category.auditRefs.length, 'renders correct number of audits');
+
+    assert.equal(categoryDOM.querySelector('.lh-gauge__wrapper--plugin'), null);
```

Comment:
a string explanation like the above asserts would be nice (or a comment, but when in Rome, etc). No plugin categories in the sample results.

Reply:
done